service_container/tags.rst

```diff
@@ -405,3 +405,92 @@ The double loop may be confusing. This is because a service can have more
 than one tag. You tag a service twice or more with the ``app.mail_transport``
 tag. The second foreach loop iterates over the ``app.mail_transport``
 tags set for the current service and gives you the attributes.
+
+Reference tagged services
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case your tag doesn't require any further additional attributes writing compiler 
+passes per tag might become tedious. A way to overcome this is is to make your compiler 
+pass more generic. The downside of this approach is you have to write and maintain
+additional code, considering you want to reuse it over multiple projects.
```

Comment:
I don't think we need so much information and we're kinda missing the complete feature here: Using tags to inject a list of services to another service. Tags can do non-injecting stuff as well. What about saying something like: (change at your free will 😄 )

 > If you use tags to inject a list of services as an argument, writing a compiler pass is a bit tedious. As this is a very common case, Symfony provides a way to inject all services tagged with a specific tag. The downside of this feature is that you can't have any custom attributes. In the example below, all services tagged with ``app.handler`` are passed in an array as the first constructor argument to the ``App\\HandlerCollection`` service:

Reply:
agree. this text is good as a blog post introducing this new feature as it explains the thought process. but it does not fit as a feature documention